Contour plot (#256)
PhiLFitters/kafe2: doc/src/parts/user_guide.rst

    from kafe2 import ContoursProfiler
    cpf = ContoursProfiler(fit)
		
By default, the one and two :math:`\sigma` contours are created. In case other values are desired,

> they can be specified in the creation of the ContoursProfiler object:

Reviewer: ```suggestion
By default, the one and two :math:`\sigma` contours are created.
Other values can be specified in the ContoursProfiler constructor:
```

    from kafe2 import ContoursProfiler
    cpf = ContoursProfiler(fit)
		

> By default, the one and two :math:`\sigma` contours are created. Other values can be

Reviewer: ```suggestion
By default, the one and two :math:`\sigma` contours are calculated. Other values can be
```

specified in the ContoursProfiler constructor:

.. code-block:: python

    from kafe2 import ContoursProfiler

>     cpf = ContoursProfiler(fit, contour_sigma_values = (1,2,3))

Reviewer: ```suggestion
    cpf = ContoursProfiler(fit, contour_sigma_values=(1, 2, 3))
```

This is I think the I think officially recommended code style.

Often confidence levels are not reported in units of :math:`\sigma`, but in percentages
(e.g. 95%). To specify e.g. the 90% and 95% confidence level contours:

.. code-block:: python

    from kafe2 import ContoursProfiler

>     cpf = ContoursProfiler(fit, contour_cl_values = (0.9, 0.95))

Reviewer: ```suggestion
    cpf = ContoursProfiler(fit, contour_cl_values=(0.9, 0.95))
```

Consult the api documentation on details how to restrict the displayed ranges
or for special plot options.

>  

Reviewer: ```suggestion

```
